input/pagecontent/5_content-sharing.md

```diff
@@ -0,0 +1,78 @@
+In addition to basic context synchronization, FHIRcast supports real-time exchange of resources between clients subscribed to the same topic.  A key concept of the content sharing events is that the content is shared in a transactional manner.
+
+The diagram below shows a series of operations beginning with a `[FHIR resource]-open` request followed by three `[FHIR resource]-update` requests.  The content in an anchor context is built up by the successive `[FHIR resource]-update` requests which contain only changes to the current state.  These changes are propagated by the Hub to all subscribed clients using `[FHIR resource]-update` events containing only the changes to be made.
+
+{% include img.html img="TransactionalUpdates.png" caption="Figure: Transactional Content Sharing Approach" %}
+
+In order to avoid lost updates and other out of sync conditions, the Hub serves as the transaction coordinator.  It fulfills this responsibility by creating a version of the content's state with each update operation.  If an operation is requested by a client which provides an incorrect version, this request is rejected.  This approach is similar to the version concurrency approach used by [FHIR versions and managing resource contention](https://www.hl7.org/fhir/http.html#concurrency).  Additionally, many of the FHIRcast content sharing concepts have similarities to the [FHIR messaging mechanisms](https://www.hl7.org/fhir/messaging.html) and where possible the approaches and structures are aligned.
+
+FHIR resources are used to convey the structured information being exchanged in `[FHIR resource]-update` operations.  However, it is possible that these resources are never persisted in a FHIR server.  During the exchange of information, the content (FHIR resource instances) is often very dynamic in nature with a user creating, modifying, and even removing information which is being exchanged.  For example, a measurement made in an imaging application could be altered many times before it is finalized and it could be removed.
+
+### Responsibilities of a FHIRcast Hub and a Subscribed Client
+
+If supporting content sharing, a FHIRcast Hub MUST fulfill additional responsibilities:
+1. Assign and maintain an anchor context's `versionId` when processing a `[FHIR resource]-open` request - the `versionId` does not need to follow any convension but must unique in the scope of a topic  
+2. Validate `[FHIR resource]-update` and `[FHIR resource]-select` requests for the correct `versionId` and reject the request if the version does not match the current `versionId` by returning a 4xx/5xx HTTP Status Code rather than updating the content or indication of selection
```

Comment:
```suggestion
2. Reject `[FHIR resource]-update` and `[FHIR resource]-select` requests if the version does not match the current `versionId` by returning a 4xx/5xx HTTP Status Code rather than updating the content or indication of selection
```
The hub doesn't actually do any validation other than reject is versionid is wrong, right?